From 1a9105129d28e21a69d4cf04696e926f065db1b6 Mon Sep 17 00:00:00 2001
From: Joshua Riek <jjriek@verizon.net>
Date: Sun, 21 Apr 2024 11:20:51 -0400
Subject: [PATCH 269/364] arm64: dts: opi5+: disable hdmirx

On the Orange Pi 5 Plus hdmirx spams system interrupts
causing a load average greater than 1.00
---
 arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts b/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
index b1c31b109cbc..cae5279bbcca 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
@@ -320,8 +320,3 @@ &spi4 {
 	assigned-clock-rates = <200000000>;
 	num-cs = <2>;
 };
-/*** 40 pins ***/
-
-&hdmirx_ctrler {
-	status = "okay";
-};
-- 
2.43.0

